lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/controls/AttributionDialog.kt

```diff
@@ -0,0 +1,66 @@
+package dev.sargunv.maplibrecompose.compose.controls
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.AlertDialogDefaults
+import androidx.compose.material3.BasicAlertDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import dev.sargunv.maplibrecompose.compose.StyleState
+
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+public fun AttributionDialog(
+  styleState: StyleState,
+  onDismiss: () -> Unit,
+  titleText: String = "Data sources",
+  buttonText: String = "OK",
+) {
+  val attributions = remember(styleState) { styleState.queryAttributionLinks() }
+  val uriHandler = LocalUriHandler.current
+
+  BasicAlertDialog(
```

Comment:
This alert dialog definition looks unnecessarily complex (with the creation of a surface with hardcoded width, hardcoded padding, text styles etc.). I only used material2 AlertDialogs so far, but looking at 

https://developer.android.com/develop/ui/compose/components/dialog#alert

it looks like you could use the `AlertDialog` composable.

Reply:
Ah, I saw `AlertDialog` was deprecated but there's an `expect fun` overload that's not deprecated. Will check it out

Reply:
https://github.com/sargunv/maplibre-compose/pull/138